src/auth/auth.service.ts

```
@@ -0,0 +1,295 @@
+    if (savedCode !== code) {
+      throw InvalidCodeException();
+    }
+
+    await this.redisService.del(
+      codeKey(normalizedEmail),
+      attemptKey(normalizedEmail),
+    );
+
+    return {
+      email: normalizedEmail,
+      verified: true
```

ESLint

[error] Insert `,`

(prettier/prettier)